docs/api/rabbitmq.com.ref.asciidoc

```diff
@@ -84,7 +84,7 @@ PersistentVolumeClaim is an embedded version of k8s.io/api/core/v1.PersistentVol
 [id="{anchor_prefix}-github-com-rabbitmq-cluster-operator-api-v1beta1-plugin"]
 ==== Plugin (string) 
 
-
+kubebuilder validating tags 'Pattern' and 'MaxLength' must be specified on string type. Alias type 'string' as 'Plugin' to specify schema validation on items of the list 'AdditionalPlugins'
 
```

Comment:
This comment doesn't make any sense. It looks like an error message from kubebuilder or `code-gen`. We should remove this (not include it for commit) or investigate and fix the root cause.

Reply:
removed